lms/djangoapps/discussion_api/api.py

```diff
@@ -276,7 +286,12 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
         "per_page": page_size,
         "text": text_search,
     }
+
     text_search_rewrite = None
+
+    if view:
+        query_params[view] = "true"
```

Comment:
What is the relationship between this method and the form validation (at what point is the forms validation done)?